packages/node/src/lib/__tests__/create-url.test.ts

```diff
@@ -0,0 +1,36 @@
+import { tryCreateFormattedUrl } from '../create-url'
+
+describe(tryCreateFormattedUrl, () => {
+  it('should create a valid url', () => {
+    expect(tryCreateFormattedUrl('http://foo.com', '/bar')).toBe(
+      'http://foo.com/bar'
+    )
+  })
+  it('should work if the path has many segments', () => {
+    expect(tryCreateFormattedUrl('http://foo.com', '/bar/baz')).toBe(
+      'http://foo.com/bar/baz'
+    )
+  })
+  it('should ignore slashes', () => {
+    const result = 'http://foo.com/bar'
+    expect(tryCreateFormattedUrl('http://foo.com/', '/bar')).toBe(result)
+    expect(tryCreateFormattedUrl('http://foo.com', 'bar')).toBe(result)
+    expect(tryCreateFormattedUrl('http://foo.com', '/bar')).toBe(result)
+    expect(tryCreateFormattedUrl('http://foo.com', '/bar')).toBe(result)
```

Comment:
I'm not sure how these 2 lines are different...

Reply:
It's subtle